src/Microsoft.AspNet.Mvc.Razor/RazorPage.cs

```diff
@@ -563,24 +554,13 @@ public virtual void WriteLiteralTo([NotNull] TextWriter writer, string value)
                     // Otherwise the value is another object (perhaps an HtmlString) and we'll ask it to format itself.
```

Comment:
Comment relates to details of `ShouldRenderAttributeValue()` and `GetStringAttributeValue()`. Mentioned special cases are not visible here.